source/includes/fact-write-blocking-requirement.rst

```diff
@@ -1,3 +1,11 @@
 To set ``enableUserWriteBlocking``, the ``mongosync`` user must have a
-role that includes the ``setUserWriteBlockMode`` and
-``bypassWriteBlockingMode`` ActionTypes.
\ No newline at end of file
+role that includes the :authaction:`setUserWriteBlockMode` and
+:authaction:`bypassWriteBlockingMode` ActionTypes. 
+
+.. note:: 
+    
+   When using ``enableUserWriteBlocking``, writes are only blocked for users
+   that do not have the :authaction:`bypassWriteBlockingMode` ActionType. Users
+   that do have this ActionType are still able to perform writes.
```

Comment:
[suggestion]

Consider updating to: "Users who have this ActionType are able to perform writes."